torchaudio/models/wavernn.py

```diff
@@ -324,3 +333,35 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
 
         # bring back channel dimension
         return x.unsqueeze(1)
+
+
+def wavernn(pretrained: bool = True, progress: bool = True, **kwargs: Any) -> WaveRNN:
```

Comment:
let's stay closer to the convention set in [here](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py): we have a helper function `_wavernn` that passes the kwargs to WaveRNN, and a particular one called `wavernn_10k_epochs_8bits_ljspeech`